src/systems/filecoin_nodes/clock/clock_subsystem.go

```diff
@@ -2,10 +2,27 @@ package clock
 
 import "time"
 
-func (_ *WallClock_I) NowUTC() Time {
+func (_ *Clock_I) UTCTime() Time {
```

Comment:
Um, why changing the `Now` interface? 

this is to make it completely obvious. it's a very useful addition in Golang to distinguish "get the time right now" from time conversions of other time values.

Reply:
maybe `NowUTCTime()` if you want